exercises/C-comparison-operators/exercise.js

```diff
@@ -5,16 +5,16 @@
   The variables should have values that match the expected results.
 */
 
-var studentCount = 16;
-var mentorCount = 9;
-var moreStudentsThanMentors; // finish this statement
+const studentCount = 16;
+const mentorCount = 9;
+const moreStudentsThanMentors = studentCount > mentorCount; // finish this statement
 
-var roomMaxCapacity = 25;
-var enoughSpaceInRoom; // finish this statement
+const roomMaxCapacity = 25;
+const enoughSpaceInRoom = studentCount + mentorCount < 25; // finish this statement
```

Comment:
This isn't correct - there are 25 people (students + mentors) so the room has space for them all. What do you need to use instead of '<'?